lib/browser/api/crash-reporter.ts

```diff
@@ -45,7 +45,7 @@ class CrashReporter {
         return bts - ats;
       });
 
-    return (reports.length > 0) ? reports[0] : null;
+    return (reports.length > 0) ? reports[0] : null!;
```

Comment:
The typings are being generated as `getLastCrashReport(): CrashReport;` with `null` missing